Example/Showcase/Classes/DomainLayer/Chat/ChatFactory.swift

```diff
@@ -12,10 +13,11 @@ class ChatFactory {
         let client = HTTPClient(host: "keys.walletconnect.com")
         let registry = KeyserverRegistryProvider(client: client)
         let relayClient = RelayClient(relayHost: relayHost, projectId: projectId, keychainStorage: keychain, socketFactory: SocketFactory())
-        return ChatClient(
+        return ChatClientFactory.create(
```

Comment:
I do not like that the integrator need to use factory
in Sign we were able to avoid this as we have a `Sign.instance` 
but we will probably add singleton wrapper for Chat as well

Reply:
yes, ChatClientFactory.create will be called in Chat singleton only